extensions/3rdparty/tabber/tabber.js

```diff
@@ -156,22 +156,24 @@ tabberObj.prototype.init = function(e)
 	this.tabs.length = 0;
 
 	/* Loop through an array of all the child nodes within our tabber element. */
-	childNodes = e.getElementsByClassName(this.classTab);
+	childNodes = e.childNodes;
 	for (i=0; i < childNodes.length; i++) {
-		/* Create a new object to save info about this tab */
-		t = {};
+		if (childNodes[i].classList.contains(this.classTab)) {
```

Comment:
Do all browsers that we support know `Array.prototype.contains`?

Reply:
Yeah, it's `[DOMTokenList](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList)` which is supported in all the browsers we support